app/bundles/CampaignBundle/Form/Type/ConfigType.php

```diff
@@ -62,6 +64,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function getName()
     {
-        return 'campaignconfig';
+        return self::class;
```

Comment:
The `getName()` method is deprecated. I think it can be removed completely in this case.